### PR TITLE
fix(FrqcyTxPayment): ensure pay_with_capacity errors

### PR DESCRIPTION
**Pull request by enddynayn**
# Goal
Ensure that when calling "pay_with_capacity," errors are propagated.
Errors are being swallowed and not propagated to the client when an extrinsic call fails.

Update benchmarks for frequency-tx-payment pallet.

### CODE REVIEW

**Review by mattheworris — APPROVED** [review]
✅

File: integration-tests/capacity/transactions.test.ts [hunk 1]
```diff
@@ -113,26 +114,21 @@ describe("Capacity Transactions", function () {
 
             // A registered provider with Capacity but no tokens associated with the
             // key should still be able to use polkadot UI to submit a capacity transaction.
-            // *All accounts will have at least an EXISTENTIAL_DEPOSIT = 1M.
-            // This test may still be valid if the txn cost is greater than 1M.
             it("pays for a transaction with available capacity and no free tokens", async function () {
                 let delegatorKeys = createKeys("delegatorKeys");
-                let _delegatorMsaId = await ExtrinsicHelper.createMsa(delegatorKeys);
+                await fundKeypair(devAccounts[0].keys, delegatorKeys, 100n * dollars);
```
Comment (shannonwells) [inline integration-tests/capacity/transactions.test.ts:119]: I don't understand the purpose of these changes in this test and above.  What was wrong with the test setup?

File: pallets/frequency-tx-payment/src/lib.rs [hunk 2]
```diff
@@ -177,11 +177,10 @@ pub mod pallet {
 		pub fn pay_with_capacity(
 			origin: OriginFor<T>,
 			call: Box<<T as Config>::RuntimeCall>,
-		) -> DispatchResult {
+		) -> DispatchResultWithPostInfo {
```
Comment (shannonwells) [inline pallets/frequency-tx-payment/src/lib.rs:180]: bingo

File: Makefile [hunk 3]
```diff
@@ -154,9 +156,6 @@ benchmarks-multi-local:
 benchmarks-capacity:
 	./scripts/run_benchmark.sh -p capacity
 
-benchmarks-frequency-tx-payment:
```
Comment (shannonwells) [inline Makefile:157]: why is “benchmarks-frequency-tx-payment” removed from the Makefile?

File: integration-tests/capacity/transactions.test.ts (hunk 1 above)
Comment (enddynayn) [reply integration-tests/capacity/transactions.test.ts:119]: The await does not belong there. It is doing nothing. It's broken.

**Review by shannonwells — APPROVED** [review]
Thanks, looks good

**Pull request merged**